objects/scienceoutpost/scienceinfobooth/scienceinfobooth.object

```diff
@@ -0,0 +1,91 @@
+    "Have a favorite weapon or armor set that you don't want to leave behind as you progress? Good news - you don't have to! Using your Tricorder's Upgrade function, you can use Upgrade modules to improve almost all Frackin' Universe weapons or armor. By doing this, you can ensure that any armor or weapon you like stays viable for the entire game!",
+    "Does an enemy seem almost invincible? It might just be resistant to the damage type you're using. Try switching things up with different weapons to find a weakness!",
+    "Don't know where to find a specific item or material? A Lab Directory can tell you if it's an output from any of your nearby machines. If that fails, check the official wiki displayed on the main menu!",
+    "Even carnivores have a reason to set up plant farms - many of the growable plants that Frackin' Universe adds can be eaten by anyone. Plus, Meatvine is a reliable source of nutrition, and many plants and their seeds can be extracted into useful materials!",
+    "Sorry we can't let you use most of the crafting and processing machines - We had to restrict access after Kevin tried to use them to create a gun that shot smaller versions of itself.",
+    "Most hazardous environmental effects can be prevented with either an immunity to that effect, or often through a high enough resistance to specific damage types.",
+    "Immunities and resistances are seperate - Even if you're immune to a planet's deadly heat, it won't reduce damage from fiery attacks unless you increase your heat resistance.",
+    "If things don't seem to be working right, first check to make sure you don't have any incompatible mods installed. If that doesn't work, post your full log file to the bug channel on the Frackin' Universe Discord.",
+    "Take your time - if you rush to advance as fast as possible, you'll miss all sorts of fun things - like these hints!",
+    "Mixing certain liquids can have unexpected results - do things right, and you can make more of almost any liquid!",
+	"Liquids will drain away if there's not a background block keeping them there - though oceans are an exception.",
+	"Don't feel like using the teleporter to descend to the planet you're orbiting? Go down far enough outside your ship, and you'll fall to the planet below!",
+	"If some of my advice doesn't make sense to you right now, don't worry about it. You never know - it might come up later!",
+	"If you ever find alien ruins with grey bricks and purple or blue glyphs, you've probably found something built by the Ancients, a long-extinct race. Apparently, many Peglaci claim that there was another race even older than them, but not much is known about them.",
+	"Speaking of, I'm actually a Peglaci - I'm just using ultrasonic vibrations to create the illusion that I'm a Hylotl.",
+	"Why am I doing that? Simple - the person who added me is garbage at pixel art, and this sprite already existed."
```

Comment:
This is mostly a joke on the fact that the sprite used is exactly the same as the info terminal at the beginning of the game. It's true that I have zero talent when it comes to pixel art, or art in general, so I'm certainly not the person to draw up new sprites for something like this. Also, thought it'd be neat to reference the Peglaci's sound-related abilities, which I believe are otherwise unrepresented so far?